test/lib/mayaUsd/render/mayaToHydra/testMayaLights.py

```diff
@@ -20,12 +20,23 @@
 import mayaUtils
 import unittest
 
+import platform
+
 class TestMayaLights(mtohUtils.MtohTestCase): #Subclassing mtohUtils.MtohTestCase to be able to call self.assertSnapshotClose
     # MayaHydraBaseTestCase.setUpClass requirement.
     _file = __file__
 
-    IMAGE_DIFF_FAIL_THRESHOLD = 0.01
-    IMAGE_DIFF_FAIL_PERCENT = 0.2
+    @property
+    def imageDiffFailThreshold():
```

Comment:
great, miss "self" as parameter?

Reply:
Yes definitely, though somehow it ran fine locally without it